cluster-provision/gocli/cmd/run.go

```diff
@@ -34,17 +34,19 @@ import (
 
 const (
 	proxySettings = `
-mkdir -p /etc/systemd/system/docker.service.d/
+curl {{.Proxy}}/ca.crt > /etc/pki/ca-trust/source/anchors/docker_registry_proxy.crt
+update-ca-trust
 
-cat <<EOT  >/etc/systemd/system/docker.service.d/proxy.conf
+mkdir -p /etc/systemd/system/crio.service.d
+cat <<EOT >/etc/systemd/system/crio.service.d/override.conf
 [Service]
 Environment="HTTP_PROXY={{.Proxy}}"
 Environment="HTTPS_PROXY={{.Proxy}}"
-Environment="NO_PROXY=localhost,127.0.0.1"
+Environment="NO_PROXY=localhost,127.0.0.1,registry,10.96.0.0/12,10.244.0.0/16,192.168.0.0/16,8.8.8.8,fd00:10:96::/112,fd00:10:244::/112,fd00::/64,2001:db8:1::1"
```

Comment:
From what I understand, the proxy configuration should only affect the traffic that is being originated from a host node, when CRIO tries to pull an image. Its not clear to me how does this proxy configuration affects pod-to-pod traffic, pod-to-service traffic, pod-to-internet traffic. In which way does the `iptables` configuration is being affected by these vars, that you had to add `8.8.8.8` to the NO_PROXY list?

Reply:
It configures a proxy, all the traffic of CRIO is going there, not only traffic from host node when trying to pull an image.
The fact that the proxy we are using is an image cache is just a special use case.
You can run it locally, and see that if you don't put for example the `registry` it will fail to load from it, because it will contact the proxy instead of the `registry`

Reply:
Rechecked several times, it seems that when running the tests not part of the conformance,
they do pass even if 8.8.8.8, but flaky sometimes, the proxy adds some delay or so so the ping timeouts.
I am removing `8.8.8.8` and `2001:db8:1::1`, as It is controversial, works basically, and not crucial when running with proxy locally.